api/zeebe/generation-strategy.js

```diff
@@ -22,15 +23,15 @@ function hackChangesetDescription(specPath) {
   // This adjustment replaces the description of the `Changeset` schema with the current description of
   //   the `UserTaskUpdateRequest.changeset` property.
   console.log("hacking changeset description...");
-  replace.sync({
-    files: `${specPath}`,
-    from: /^      description: A map of changes.$/m,
-    to: `      description: |
-        JSON object with changed task attribute values.
 
+  const content = fs.readFileSync(specPath, "utf8");
+
+  const updated = content.replace(
+    /^ {6}description: A map of changes\.$/m,
```

Comment:
is this regex change intentional? (the `from` change)

Reply:
Good point, I think it makes sense to keep it as is, will revert.